src/components/chatroom/chat/chat.chatroom.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from "react";
+import { useParams } from "react-router-dom";
+import "./chat.chatroom.scss";
+import Form from "./form/form.chatroom";
+import ChatLog from "./chatlog/chatlog.chatroom";
+
+import Header from "./header/header.chatroom";
+
+function Chat({ chats, setChats }) {
+  let { id } = useParams();
+
+  let [userId, setUserId] = useState(0);
```

Comment:
```suggestion
  const { id } = useParams();

  const [userId, setUserId] = useState(0);
```
`let`도 필수불가결한 경우(사실 리액트의 state 관리를 생각해보면 거의 그런 경우는 없습니다)를 제외하면 지양하는게 좋습니다